misc/liveindextest.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+#
+# usage:
+#  python3 misc/liveindextest.py
+#
+# Requires go-algorand to be checked out on GOPATH.
+# Requires local postgresql and `createdb` `dropdb` standard utils.
```

Comment:
Is it safe to assume that this is only true when a connection string isn't provided?

Reply:
Yes. I guess I only documented the default case. An externally managed database with passed in connection string is neither created nor destroyed within this script.